### PR TITLE
feat: enable logging pcs params

### PR DESCRIPTION
**Pull request by chokobole**
# Description

This PR enables logging PCS params. This help us reducing time-consuming unsafe setup.

### CODE REVIEW

**Review by dongchangYoo — APPROVED** [review]
LGTM

**Review by Insun35 — APPROVED** [review]
LGTM

File: tachyon/c/math/polynomials/constants.h [hunk 1]
```diff
@@ -8,7 +8,7 @@ namespace tachyon::c::math {
 
 // NOTE(chokobole): We set |kMaxDegree| to |SIZE_MAX| on purpose to avoid
```
Comment (ashjeong) [inline tachyon/c/math/polynomials/constants.h:9]:
```Suggestion
// NOTE(chokobole): We set |kMaxDegree| to |SIZE_MAX| - 1 on purpose to avoid
```

Comment (ashjeong) [reply tachyon/c/math/polynomials/constants.h:9]: I remember seeing this comment in multiple files. Is this the only one we need to change? If not, then is it not better to standardize the same named variables across files?

**Review by TomTaehoonKim — APPROVED** [review]
LGTM

File: tachyon/c/math/polynomials/constants.h (hunk 1 above)
Comment (chokobole) [reply tachyon/c/math/polynomials/constants.h:9]: Yes I changed [here](https://github.com/kroma-network/tachyon/pull/277/commits/146ad120c168b7b37faf0cd9bdd658d7b070e4dd)

**Review by ashjeong — APPROVED** [review]
LGTM

**Pull request merged**